enterprise/core-edge/src/main/java/org/neo4j/commandline/dbms/UnbindFromClusterCommand.java

```diff
@@ -0,0 +1,193 @@
+import org.neo4j.commandline.admin.IncorrectUsage;
+import org.neo4j.commandline.admin.OutsideWorld;
+import org.neo4j.dbms.DatabaseManagementSystemSettings;
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.helpers.Args;
+import org.neo4j.io.fs.FileUtils;
+import org.neo4j.kernel.StoreLockException;
+import org.neo4j.kernel.configuration.Config;
+import org.neo4j.kernel.impl.util.Converters;
+import org.neo4j.kernel.impl.util.Validators;
+import org.neo4j.server.configuration.ConfigLoader;
+
+import static java.lang.String.format;
+import static java.nio.file.Files.exists;
+import static org.neo4j.coreedge.core.EnterpriseCoreEditionModule.CLUSTER_STATE_DIRECTORY_NAME;
+
+public class UnbindFromClusterCommand implements AdminCommand
+{
+
+    private StoreLockChecker storeLockChecker;
+
+    public static class Provider extends AdminCommand.Provider
+    {
+        public Provider()
+        {
+            super( "unbind" );
+        }
+
+        @Override
+        public Optional<String> arguments()
+        {
+            return Optional.of( "[--database=<name>] " );
+        }
+
+        @Override
+        public String description()
+        {
+            return "Removes cluster state data from the specified database making it suitable for use non-Core-Edge " +
+                    "databases";
+        }
+
+        @Override
+        public AdminCommand create( Path homeDir, Path configDir, OutsideWorld outsideWorld )
+        {
+            return new UnbindFromClusterCommand( homeDir, configDir );
+        }
+    }
+
+    private Path homeDir;
+    private Path configDir;
+
+    UnbindFromClusterCommand( Path homeDir, Path configDir )
+    {
+        this.homeDir = homeDir;
+        this.configDir = configDir;
+        this.storeLockChecker = new StoreLockChecker();
+    }
+
+    @Override
+    public void execute( String[] args ) throws IncorrectUsage, CommandFailed
+    {
+        Args parsedArgs = Args.parse( args );
+
+        try
+        {
+            Config config = loadNeo4jConfig( homeDir, configDir, databaseNameFrom( parsedArgs ) );
+            Path pathToSpecificDatabase = config.get( DatabaseManagementSystemSettings.database_path ).toPath();
+
+            Validators.CONTAINS_EXISTING_DATABASE.validate( pathToSpecificDatabase.toFile() );
+            confirmClusterStateDirectoryExists( Paths.get( pathToSpecificDatabase.toString(), "cluster-state" ) );
+
+            try ( Closeable ignored = storeLockChecker.withLock( pathToSpecificDatabase ) )
+            {
+                deleteClusterStateIn( clusterStateFrom( pathToSpecificDatabase ) );
+            }
+            catch ( IOException e )
+            {
+                wrapIOException( e );
+            }
+
+        }
+        catch ( StoreLockException e )
+        {
+            throw new CommandFailed(
+                    "Database is currently locked. Please check that no other Neo4j instance using it.", e );
```

Comment:
"Please check that no other Neo4j instance _is_ using it."
Or, "Please check that it is not being used by another Neo4j instance."